internal/service/service_integration/resource_service_integration_test.go

```diff
@@ -627,3 +627,57 @@ func TestAccAivenServiceIntegration_clickhouse_kafka_user_config_creates(t *test
 		},
 	})
 }
+
+func testAccServiceIntegrationClickhousePostgresUserConfig(prefix, project string) string {
+	return fmt.Sprintf(`
+data "aiven_project" "project" {
+  project = %[2]q
+}
+
+resource "aiven_pg" "pg" {
+  project      = data.aiven_project.project.project
+  cloud_name   = "google-europe-west1"
+  plan         = "startup-4"
+  service_name = "%[1]s-pg"
+}
+
+resource "aiven_clickhouse" "clickhouse" {
+  project                 = data.aiven_project.project.project
+  cloud_name              = "google-europe-west1"
+  plan                    = "startup-beta-8"
+  service_name            = "%[1]s-clickhouse"
+  maintenance_window_dow  = "monday"
+  maintenance_window_time = "10:00:00"
+}
+
+resource "aiven_service_integration" "clickhouse_pg_source" {
+  project                  = data.aiven_project.project.project
+  integration_type         = "clickhouse_postgresql"
+  source_service_name      = aiven_pg.pg.service_name
+  destination_service_name = aiven_clickhouse.clickhouse.service_name
+}
+`, prefix, project)
+}
+
+func TestAccAivenServiceIntegration_clickhouse_postgres_user_config_creates(t *testing.T) {
+	project := os.Getenv("AIVEN_PROJECT_NAME")
+	prefix := "test-acc-" + acctest.RandString(7)
+	resourceName := "aiven_service_integration.clickhouse_pg_source"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acc.TestAccPreCheck(t) },
+		ProviderFactories: acc.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckAivenServiceIntegrationResourceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config:             testAccServiceIntegrationClickhousePostgresUserConfig(prefix, project),
+				ExpectNonEmptyPlan: true,
```

Comment:
what do you think if instead of doing `ExpectNonEmptyPlan` we will have the `database` section explicitly defined in the test?

I believe, this would be also great for example usage of this integration

Reply:
Done.

```
=== RUN   TestAccAivenServiceIntegration_clickhouse_postgres_user_config_creates
=== PAUSE TestAccAivenServiceIntegration_clickhouse_postgres_user_config_creates
=== CONT  TestAccAivenServiceIntegration_clickhouse_postgres_user_config_creates
--- PASS: TestAccAivenServiceIntegration_clickhouse_postgres_user_config_creates (281.63s)
PASS

Process finished with the exit code 0
```